mkdocs.yaml

```diff
@@ -55,7 +55,9 @@ plugins:
       regex:
         - ^(?!(.*/)?assets/).*\.(?!(.*\.)?md|(.*\.)?svg|(.*\.)?png|(.*\.)?gif|(.*\.)?jpg).*$
         - ^(.*/)?[^.]*$
-  - macros
+  - macros:
+      include_yaml:
+        - autoware_interfaces: yaml/autoware-interfaces.yaml
```

Comment:
This setting is for autoware-documentation repositories only. Should I create a dedicated file for the sync source?

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/3980